conda/conda-recipes/azure-mgmt/meta.yaml

```diff
@@ -271,11 +276,6 @@ test:
     - azure.mgmt.datafactory.aio.operations
     - azure.mgmt.datafactory.models
     - azure.mgmt.datafactory.operations
-    - azure.mgmt.datalake.store
```

Comment:
Intentional yeah?

Reply:
I think it it may be, given the lack of `azure-mgmt-datalake` package in the mgmt package list below.

Reply:
Yes. They yanked the only GAed version.

https://pypi.org/project/azure-mgmt-datalake-store/#history